app/models/report_template.rb

```diff
@@ -13,7 +15,11 @@ def serialization_params=(value)
 
   def serialization_params
     value = read_attribute :serialization_params
-    value.nil? ? {} : Marshal.load(value)
+    begin
+      value.nil? ? {} : Marshal.load(value)
+    rescue TypeError

```

Comment:
Como boa prática peço que remova a alteração. Se for necessário de fato o ideal é criar em seu projeto uma subclasse desta classe que faça isto e a use somente no seu projeto. Como se trata de uma biblioteca usada em vários projetos não temos como aceitar esta mudança na classe comum.